docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+services:
+  todoapp:
+    image: todoapp:1.0.0
```

Comment:
The `image` field specifies `todoapp:1.0.0`. Ensure that this image is available locally or in a Docker registry. If you intend to build the image from the Dockerfile, you might not need this field.